### PR TITLE
Create the museum

### PR DESCRIPTION
**Pull request by FeixLiu**

### CODE REVIEW

**Comment by FeixLiu** [comment]
> Looks good but you have a lot of commits in this PR. It is better to only have 1 commit per PR. Please squash your commits

I've squash all the commits together! It's really not an easy work :)

File: 01-python/leisure/museum.py [hunk 1]
```diff
@@ -1,3 +1,13 @@
-def draw_museum():
-    print("museum not found")
+
+def draw_museum(items=None, name=None, location=None):
+    if items is None:
+        items = ["Mona Lisa"]
```
Comment (gallettilance) [inline 01-python/leisure/museum.py]: Please set these values as the default in the parameters of the function

File: 01-python/leisure/museum.py [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+    if name is None:
+        name = "FMA"
```
Comment (gallettilance) [inline 01-python/leisure/museum.py]: here too

File: 01-python/leisure/museum.py [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+    if location is None:
+        location = "Boston"
```
Comment (gallettilance) [inline 01-python/leisure/museum.py]: here too

File: 01-python/leisure/museum.py [hunk 4]
```diff
@@ -1,3 +1,13 @@
...
+    print("Welcome to the Museum " + name + " which is located in " + location)
```
Comment (gallettilance) [inline 01-python/leisure/museum.py]: if there are no items, no name, and no location, should the function print a museum?

Comment (FeixLiu) [reply 01-python/leisure/museum.py]: Make some updates for these issues. Thanks a lot.

Comment (FeixLiu) [reply 01-python/leisure/museum.py]: > if there are no items, no name, and no location, should the function print a museum?

Make some updates for these issues. Thanks a lot.

**Pull request closed without merging**